notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Nit: BigQuery DataFrames not BigFrames</p><p> </p><p>Nit: For some reason, pandas always uses lower-case "p", even when at the start of a sentence. See: <a href="https://pandas.pydata.org" target="_blank">https://pandas.pydata.org/</a></p><p> </p><p>Nit: Use 2nd person. "We'll" is not recommended.</p><p> </p><p>Suggestion:</p><p> </p><p>This section provides practical code examples illustrating how to use struct columns with BigQuery DataFrames.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Thanks!</p><p> </p><p> </p><p> </p>